services/tenant-ui/frontend/src/plugins/i18n/locales/en.json

```diff
@@ -326,6 +326,7 @@
     "imageUrl": "Image URL",
     "issuer": "Issuer",
     "name": "Name",
+    "contact": "Email",
```

Comment:
Think I'd make this "Contact Email" explicitly so if one of multiple users see the profile it's a little clearer it's the contact email for the Tenant and not their personal email.